src/signalSTM.h

```diff
@@ -0,0 +1,354 @@
+#pragma once
+
+#if !defined(__AVR__) && defined(ARDUINO_MAPLEMINI_F103CB) || defined(STM32F103xB)
+#include "compile_config.h"
+
+/*
+
+*   developer:
+*   2020 S.Butzek, HomeAutoUser, elektron-bbs
+
+*   Integration for compatibility with a similar project by Ralf9
+
+*   tested hardware:
+*    - Maple Mini STM32F103CBT6
+
+*/
+
+#define PROGNAME               " SIGNALduino_STM32 "
+#define PROGVERS               "3.4.0-dev_20200626"
+#define VERSION_1               0x33
+#define VERSION_2               0x1d
+
+#define BAUDRATE               115200
+#define FIFO_LENGTH            170
+#define defSelRadio 1                           // variant -> Circuit board for 4 cc110x - standard value 1 = B
+const uint8_t pinReceive[] = {11, 18, 16, 14};  // variant -> Circuit board for 4 cc110x
+uint8_t radionr = defSelRadio;                  // variant -> Circuit board for 4 cc110x
```

Comment:
Ist das für irgendwas gut, das als änderbare Vartiable zu haben? Mir scheint, dass es immer den Wert von Macro defSelRadio  besitzt.

Reply:
Ob das gut ist weiß ich nicht. Ich übernahm dies so und wenn ich richtig in den Code geschaut habe, so wird einmal die Variable radionr genutzt welche von 1 - 4 springen kann bei mehreren Empfängern und die Variable defSelRadio mit dem Wert 1 wird genutzt um einen Standard zu definieren.